generator/index.js

```diff
@@ -9,9 +9,13 @@ module.exports = (api, options) => {
   api.extendPackage(pkg)
 
   let buefyLines = `\nimport Buefy from 'buefy'`
-  if (options.css) {
+  if (options.includeBuefy === 'css') {
     buefyLines += `\nimport 'buefy/lib/buefy.css'`
   }
+  else if (options.includeBuefy === 'scss') {
+    api.render('./templates/scss')
+    buefyLines += `\nimport './scss/app.scss'`
```

Comment:
I think here we could use `buefy/src/scss/buefy.scss` and remove `api.render(...`

Reply:
My intent with adding the extra scss files in a template is to allow customization of Bulma and Buefy in the resulting project.  In my projects I almost never use the default color scheme or settings.  I built the app.scss and _variables.scss by combining customization method in the Buefy docs and the customization methohd in the Bulma docs.  Having all the variables defined in the files may be a little overkill.  What are your thoughts?

Reply:
it makes sense now.

Reply:
Then change the `includeBuefy` property to `style` please